app/library/Utils.php

```diff
@@ -45,10 +44,8 @@ public function getDocsUrl($lang)
         $return    = 'en';
         $languages = $this->config->get('doc_languages')->toArray();
 
-        if (true !== empty($lang)) {
-            if (true === array_key_exists($lang, $languages)) {
-                $return = $lang;
-            }
+        if (true !== empty($lang) && true === array_key_exists($lang, $languages)) {
```

Comment:
Sure, BTW the CONTRIBUTING.md needs an update. I was not able to figure the exact branch to make this pull request to. If you could update it, would be a big help to report issues as well.